db/migrate/20211013092406_add_normalized_email_to_user_email.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class AddNormalizedEmailToUserEmail < ActiveRecord::Migration[6.1]
+  def change
+    add_column :user_emails, :normalized_email, :string
+    execute "CREATE INDEX index_user_emails_on_normalized_email ON user_emails (LOWER(normalized_email))"
```

Comment:
Should this be a `UNIQUE` index?

Reply:
I think that if this index was `UNIQUE` then it would be like the site setting was always true.

Reply:
Good point 👍